.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Mental note: Describe how this and other setups related to stand-alone application works in README.

Reply:
done! thanks.